src/db_options.rs

```diff
@@ -394,7 +396,9 @@ unsafe impl Sync for CuckooTableOptions {}
 unsafe impl Sync for ReadOptions {}
 unsafe impl Sync for IngestExternalFileOptions {}
 unsafe impl Sync for Cache {}
+unsafe impl Sync for CacheWrapper {}
 unsafe impl Sync for Env {}
```

Comment:
Looks like `Env` will now automatically be `Send + Sync` (https://doc.rust-lang.org/std/sync/struct.Arc.html#impl-Send). Same for `Cache`.